src/NUnitFramework/framework/Internal/Tests/TestMethod.cs

```diff
@@ -76,17 +78,19 @@ internal object ExpectedResult
         {
             get { return parms != null ? parms.ExpectedResult : null; }
         }
-
-        internal object[] Arguments
-        {
-            get { return parms != null ? parms.Arguments : null; }
-        }
-
         #endregion
 
         #region Test Overrides
 
         /// <summary>
+        /// The arguments to use in executing the test method.
+        /// </summary>
+        public override object[] Arguments
+        {
+            get { return parms != null ? parms.Arguments : NoArguments; }
```

Comment:
(Explanation to the above comment: this property previously returned null if `parms == null`, rather than `NoArguments`. The diff doesn't show that very well!)